docs/dev/agent_dev_env.md

```diff
@@ -47,39 +85,27 @@ file and `pip`:
 pip install -r requirements.txt
 ```
 
-This procedure ensures you not only get the correct version of invoke, but
+This procedure ensures you not only get the correct version of `invoke`, but
 also any additional python dependencies our development workflow may require,
 at their expected versions.
-It will also pull other handy development tools/deps (reno, or docker).
+It will also pull other handy development tools/deps (`reno`, or `docker`).
 
 Tasks are usually parameterized and Invoke comes with some default values that
 are used in the official build. Such values are listed in the `invoke.yaml`
 file at the root of this repo and can be overridden by setting `INVOKE_*` env
 variables (see Invoke docs for more details).
 
-
-### Note
-
-We don't want to pollute your system-wide python installation, so a python virtual
-environment is recommended (though optional). It will help keep an isolated development
-environment and ensure a clean system python.
-
-- Install the virtualenv module:
-```pip2 install virtualenv```
-- Create the virtual environment:
-```virtualenv $GOPATH/src/github.com/DataDog/datadog-agent/venv```
-- Specify the path when building the agent:
-```invoke agent.build --python-home-2=$GOPATH/src/github.com/DataDog/datadog-agent/venv```
-
-If you are using python 3 instead (or switching between python versions), you can also
-add `--python-home-3=<path>` pointing to a python3 virtual environment.
-
 ## Golang
 
-You must install [go](https://golang.org/doc/install) version 1.11.5 or above. Make
-sure that `$GOPATH/bin` is in your `$PATH` otherwise Invoke cannot use any
+You must [install Golang](https://golang.org/doc/install) version `1.14`. Make
+sure that `$GOPATH/bin` is in your `$PATH` otherwise `invoke` cannot use any
 additional tool it might need.
 
+**Please note that versions of Golang that aren't an exact match to the version
+specified in our build image [here](https://github.com/DataDog/datadog-agent/blob/master/.circleci/images/builder/Dockerfile#L1)
```

Comment:
We use several build images, this one works but I think we should reword the sentence 
```suggestion
specified in our build images (see e.g. [here](https://github.com/DataDog/datadog-agent/blob/master/.circleci/images/builder/Dockerfile#L1))
```